src/uas/UAS.cc

```diff
@@ -147,22 +147,20 @@ UAS::UAS(MAVLinkProtocol* protocol, Vehicle* vehicle) : UASInterface(),
     blockHomePositionChanges(false),
     receivedMode(false),
 
-    // Initialize HIL sensor noise variances to 0.  If user wants corrupted sensor values they will need to set them
-    // Note variances calculated from flight case from this log: http://dash.oznet.ch/view/MRjW8NUNYQSuSZkbn8dEjY
-    // TODO: calibrate stand-still pixhawk variances
-    xacc_var(1.2914f),
-    yacc_var(0.7048f),
-    zacc_var(1.9577f),
-    rollspeed_var(0.8126f),
-    pitchspeed_var(0.6145f),
-    yawspeed_var(0.5852f),
-    xmag_var(0.4786f),
-    ymag_var(0.4566f),
-    zmag_var(0.3333f),
-    abs_pressure_var(1.1604f),
-    diff_pressure_var(1.1604f),
-    pressure_alt_var(1.1604f),
-    temperature_var(1.4290f),
+    // measured pixhawk sensor noise variances
+    xacc_var(1e-4),
+    yacc_var(1e-4),
+    zacc_var(2e-4),
+    rollspeed_var(2.5e-7),	// mpu6000 gyro total RMS noise spec: 2.5e-3 deg/sec = 8.7e-4 rad/sec => variance 7.6e-7
+    pitchspeed_var(2.5e-7),
+    yawspeed_var(2.5e-7),
+    xmag_var(4e-6),
+    ymag_var(4e-6),
+    zmag_var(3e-6),
+    abs_pressure_var(5e-4),
+    diff_pressure_var(1.0f),

```

Comment:
You haven't really reduced the differential pressure variance, its non-surprising that you end up with terrible results.